test/rejected.js

```diff
@@ -0,0 +1 @@
+Promise.reject("anything");
```

Comment:
What does this file do? I don't think you meant to commit this?

Reply:
That was research for gh-2988, and I didn't mean to commit it. Sorry about that; too many plates spinning. I've removed it.

Reply:
No problem!